2-mandatory/3-groceries.js

```diff
@@ -51,35 +66,59 @@ let numberOfItemsPerWeek = {
   saturday: 0,
   sunday: 0,
 };
-
+for (let day in weeklyMealPlan) {
```

Comment:
Again, really nice Laleh! I noticed you are using a lot of `for...in` loops, which is great! I personally use `Object.keys` or `Object.values` and `Object.entries` for my loops. There is no correct or incorrect way, but usually using `Object...` means you can iterate over arrays and retain immutability. As such, you can achive the same thing by doing
```
Object.keys(weeklyMealPlan).forEach((day) => numberOfItemsPerWeek[day] = weeklyMealPlan[day].length);
```
Again, both are correct, but I just want to make you aware of each method :)

Reply:
oh very helpful ,thanks